core/controllers/topic_editor.py

```diff
@@ -90,37 +88,6 @@ def post(self, topic_id):
         })
 
 
```

Comment:
Again, why are we removing this handler?

Reply:
This is moved to a unified handler in question_list.py.